source/modulo_components/test/python/test_component_interface.py

```diff
@@ -21,6 +21,22 @@ def component_interface(ros_context):
     yield ComponentInterface('component_interface')
 
 
+def test_rate_period_parameters(ros_context):
+    component_interface = ComponentInterface('component_interface')
+    assert component_interface.get_parameter_value("rate") == 10
+    assert component_interface.get_parameter_value("period") == 0.1
+
+    parameter_overrides=[rclpy.Parameter("period", value=0.01)]
+    component_interface = ComponentInterface('component_interface', parameter_overrides=parameter_overrides)
+    assert component_interface.get_parameter_value("rate") == 100
+    assert component_interface.get_parameter_value("period") == 0.01
+
+    parameter_overrides=[rclpy.Parameter("rate", value=200), rclpy.Parameter("period", value=0.01)]
+    component_interface = ComponentInterface('component_interface', parameter_overrides=parameter_overrides)
+    assert component_interface.get_parameter_value("rate") == 200
+    assert component_interface.get_parameter_value("period") == 0.005
+
```

Comment:
for completeness, would be nice to do one more case with just rate and no period